css/style.css

```diff
@@ -0,0 +1,327 @@
+}
+
+.copy-right {
+  position: absolute;
+  font-size: 10px;
+  bottom: 10px;
+  left: 0;
+}
+.main2-svg-link {
+  width: 100px;
+  height: 20px;
+}
+
+.header-main-3 svg,
+.header-main-2 svg {
+  padding-left: 10%;
+  transform: scale(1.5);
+}
+
+.header-main-3 svg {
+  margin-left: 30px;
+  height: 20px;
+  width: 80px;
+  margin-top: 0;
+}
+
+.header-bottom-menu li a {
+  font-size: 14px;
+}
+
+.header-top-menu li a {
+  font-size: 14px;
+}
+
+.border-3 {
+  border-bottom: double;
+  border-width: 6px solid black;
+  width: auto;
+  margin-bottom: 5px;
+}
+
+ .debate-border {
+  border-bottom: 3px solid #d9d9d9;
+}
+
+.card-img-top {
+  position: relative;
+}
+
+section.main-content {
+  z-index: -10;
+  overflow-x: hidden;
+}
+
+.feature-stories img,
+.top-stories img {
+  height: 400px;
+}
+
+.my-turn img,
+.cult-and-tra img {
+  max-height: 144px;
+  width: 144px;
+}
+
+.my-turn h5 a,
+.cult-and-tra h5 a {
+  font: bold 17px/21px "Roboto Condensed", sans-serif;
+  color: black;
+}
+
+.opinion-img > div,
+.opinion-img-sm .signup {
+  margin: 0 auto;
+  max-width: 300px;
+}
+
+.more-stories img,
+.latest-stories img {
+  max-height: 95px;
+  min-width: 95px;
+}
+
+.debate img {
+  min-width: 70px;
+  height: 70px;
+}
+
+.opinion-img img {
+  height: 200px;
+}
+
+.signup .icon-n {
+  width: 39px;
+  height: 38px;
+  background: url(https://web.archive.org/web/20210120125445im_/https://g.newsweek.com/www/images/logo-n1.svg) center/contain no-repeat #f72210;
+}
+
+.signup h6 {
+  font-size: 15px;
+}
+
+.debate-item {
+  width: 50%;
+  border-top: 4px solid rgb(201, 177, 177);
+  border-bottom: 3px double rgb(201, 177, 177);
+}
+
+.opinion-img > div,
+.opinion-img-sm .signup {
+  margin: 0 auto;
+  max-width: 300px;
+}
+
+.opinion-img img {
+  height: 200px;
+}
+
+  /* MEDIA QUERIES */
+@media screen and (max-width: 991.99px) {
+  .container-fluid {
+    padding: 0;
+    margin: 0;
+  }
+  .header-main-2 svg {
+    transform: scale(1.05);
+    padding-left: 80px;
+  }
+    .footer-icon {
+    padding-top: 30px;
+  }
+  section.main-content {
+    margin-top: 0;
+  }
+  .feature-stories img {
+    max-width: 100%;
+    height: 230px;
+  }
+  
+    .top-stories img {
+      height: 275px;
+    }
+  }
+
+@media screen and (min-width: 992px) {
+  .header-main-1-lg > div {
+    padding-top: 40px;
+  }
+
+  
+.top-stories-sec img {
+  height: 200px;
+}
+}
+
+.header-bottom-menu li a,
+.footer-edition li a,
+.footer-contact li a {
+  font-size: 14px;
+}
+
+@media screen and (min-width: 992.99px) {
+  .header-main-1-lg > div {
+    padding-top: 40px;
+  }
+
+  .top-stories-sec img {
+    height: 300px;
+  }
+}
+
+@media screen and (max-width: 767.99px) {
+ .header-main-2 svg {
+  margin-left: -30px;
+  height: 30px;
+  width: 250px;
+  margin-top: 25px;
+  position: static;
+}
+
+  .header-main-3 svg {
+  margin-bottom: 10px;
+}
+
+.header-main-2 img,
+.header-main-3 img {
+  width: 40px;
+  height: 40px;
+}
+
+.login {
+  display: none;
+}
```

Comment:
- [ ] This can be done with the bootstrap class `d-md-none`

Reply:
Thank you for your review, really helpful we have implemented the changes and submitted them for another re-review, we hope we get approval soon if not we will be on improvement leave .